services-transport-rsocket/src/main/java/io/scalecube/services/transport/rsocket/ExtendedEpollEventLoopGroup.java

```diff
@@ -67,7 +71,7 @@ protected EventLoop newChild(Executor executor, Object... args) throws Exception
 
   @Override
   public ChannelFuture register(Channel channel) {
-    EventExecutor eventExecutor = eventExecutorChooser.getEventExecutor(channel, iterator());
+    EventExecutor eventExecutor = eventExecutorChooser.apply(channel, iterator());
```

Comment:
If we made it `EventLoop eventLoop = eventLoopChoose.apply(...)` then we wouldn't need to cast
 on [line 76](https://github.com/scalecube/scalecube-services/pull/452/files#diff-05eda617970dc43282904f3db9bd5651R76)